indra/tools/reading/read_files.py

```diff
@@ -1,28 +1,46 @@
 """Read a list of files located in your local directory."""
-from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
-
-import logging
+import json
 import pickle
 import random
+import logging
+from os import path, listdir
 
-logger = logging.getLogger('indra.tools.reading.read_files')
-
+from indra.tools.reading.readers.core import dump_readings
 from indra.tools.reading.util.script_tools import get_parser
-from indra.tools.reading.readers import _get_dir, get_reader_classes, Content
+from indra.tools.reading.readers import get_dir, get_reader_classes, Content
+
+logger = logging.getLogger(__name__)
```

Comment:
The logger path was set explicitly here because this script will typically be run as `__main__`